packages/cardano-services/src/Asset/AssetHttpService.ts

```diff
@@ -0,0 +1,67 @@
+import * as OpenApiValidator from 'express-openapi-validator';
+import { CardanoTokenRegistry } from './CardanoTokenRegistry';
+import { DbSyncAssetProvider } from './DbSyncAssetProvider';
+import { DbSyncNftMetadataService } from './DbSyncNftMetadataService';
+import { HttpService } from '../Http';
+import { Logger } from 'ts-log';
+import { Pool } from 'pg';
+import { ServiceNames } from '../Program';
+import { createDbSyncMetadataService } from '../Metadata';
+import { providerHandler } from '../util';
+import express from 'express';
+import path from 'path';
+
+/**
+ * Dependencies that are need to create AssetHttpService
+ */
+export interface AssetHttpServiceDependencies {
+  /**
+   * The db-sync database Pgpool
+   */
+  db: Pool;
+
+  /**
+   * The logger object
+   */
+  logger: Logger;
```

Comment:
```suggestion
  logger: Logger;
  assetProvider: AssetProvider;
```

The concrete asset provider class should be instantiated and passed in here as a dependency. This is an example of where it will happen, with a provider that has an additional service needed:

https://github.com/input-output-hk/cardano-js-sdk/blob/72d83fb203d3dc417f8acc3894d3db2a398513fb/packages/cardano-services/src/Program/loadHttpServer.ts#L81-L93

Reply:
f872fec9f should resolve this.